backend/repository/handler.go

```diff
@@ -386,6 +388,44 @@ func (h *Handler) ThriftModuleToCode(w http.ResponseWriter, r *http.Request, ps
 	h.WriteJSON(w, http.StatusOK, &thriftToCodeResponse{Content: code})
 }
 
+type thriftCodeToMethodsRequest struct {
+	RawCode string `json:"rawCode"`
+}
+
+type thriftCodeToMethodsResponse struct {
+	Functions []string `json:"functions"`
+}
+
+// ThriftCodeToMethods returns a list of method names for a given thrift file path
+func (h *Handler) ThriftCodeToMethods(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	req := &thriftCodeToMethodsRequest{}
+
+	_, err := UnmarshalJSONBody(r, req)
+	if err != nil {
+		h.WriteErrorResponse(w, http.StatusBadRequest, errors.Wrap(err, "Raw code failed unmarshalling."))
```

Comment:
The convention for the wrap message is starting with lower case with no punctuation at the end. https://godoc.org/github.com/pkg/errors#ex-Wrap